sopel/modules/currency.py

```diff
@@ -2,109 +2,196 @@
 """
 currency.py - Sopel Currency Conversion Module
 Copyright 2013, Elsie Powell, embolalia.com
+Copyright 2019 Mikkel Jeppesen
 Licensed under the Eiffel Forum License 2.
 
 https://sopel.chat
 """
+
```

Comment:
We pretty well never put a blank line between the module docstring and future imports.
```suggestion
```